message/message.go

```diff
@@ -16,6 +16,15 @@ import (
 const (
 	// metaKey is a prefix used to access the meta field in a Message.
 	metaKey = "meta "
+
+	// IsControl indicates that the message is a control message.
+	IsControl = iota
```

Comment:
Style: It might be more clear that these are part of the flag system if their names are: `FlagIsControl`, `FlagSkipNullValues`, etc?

Reply:
I was avoiding stutter, like:
```go
msg.HasFlag(message.FlagIsControl)
```

The Uber style guide has what looks like [a good solution to this problem](https://github.com/uber-go/guide/blob/master/style.md#start-enums-at-one), so I added that.